cors/basic.htm

```diff
@@ -6,48 +6,43 @@
 
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
+<script src=/common/util.js></script>
 <script src=support.js?pipe=sub></script>
 <div id=log></div>
 
 <script>
-
-var counter = 0;
-
 function cors(desc, scheme, subdomain, port) {
-    if (!scheme) {
-        var url = "";
-    } else {
-        if (!port) {
-            port = location.port;
-        }
-        var url = scheme + "://" + (subdomain ? subdomain + "." : "") + location.hostname + ":" + port + dirname(location.pathname)
+    port = port || location.port;
+    subdomain = subdomain || "";
```

Comment:
Nit: you can use `port = location.port` and `subdomain = ""` in the function declaration directly these days I think, that might be nicer.

Reply:
Done.